woocommerce/woocommerce-ios
# Add simple screenshots

**To Test**
1) Download the [screenshots.zip](https://github.com/woocommerce/woocommerce-ios/files/3667580/screenshots.zip) archive, and unzip it in the `fastlane` directory. The directory structure is required to generate promo screenshots (this may be a good thing to change later).
2) Run `bundle install --with screenshots && bundle exec fastlane create_promo_screenshots`, and if prompted, confirm that you'd like to overwrite existing promo screenshots.
3) Ensure that the created screenshots look ok.
